app.js

```diff
@@ -43,6 +43,9 @@ app.set('view engine', 'ejs');
 // pass the user data from session to template global variables
 app.use((req, res, next) => {
   res.locals.user = req.session.user;
+  res.locals.accessToken = req.session.accessToken;
+  res.locals.idToken = req.session.idToken;
```

Comment:
adding accessToken and idToken in the locals variable should not be needed. Plz remove.